src/main/java/hu/oe/nik/szfmv/automatedcar/visualization/CourseDisplay.java

```diff
@@ -72,7 +72,7 @@ private void drawObjects(Graphics2D g2d, World world) {
 
         for (WorldObject object : world.getWorldObjects()) {
             AffineTransform t = new AffineTransform();
-            t.translate(object.getX(), object.getY());
+            t.translate(object.getPosX(), object.getPosY());
```

Comment:
Ez is változott már menet közben. Figyeljünk, hogy ne nyúljunk át egymás kódrészeibe, mert ez biztosan conflictot fog okozni majd.

Reply:
szerintem ismételten csak azért, hogy ne errorozzon, majd merge conflictként úgy is keletkezni fog és ott a tietek lesz kiválasztva